src/compile/axis/properties.ts

```diff
@@ -2,7 +2,7 @@ import {Align, AxisOrient, Orient, ScaleType, SignalRef} from 'vega';
 import {isArray} from 'vega-util';
 import {Axis} from '../../axis';
 import {isBinned, isBinning} from '../../bin';
-import {PositionScaleChannel, X} from '../../channel';
+import {PositionScaleChannel, X, X2, Y, Y2} from '../../channel';
```

Comment:
Please either revert this file or rebase it to https://github.com/vega/vega-lite/pull/6312.

Reply:
actually https://github.com/vega/vega-lite/pull/6372

Reply:
Hopefully it's easy enough to resolve the conflicts here. I am happy to do the rebase if needed.